### PR TITLE
Resource ownership

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Adding the resource ownership explanation

## Deploy preview link

http://pr-373.amplify-central.opendocs-builder.pcloud.axway.int/docs/integrate_with_central/api_server/#ownership-and-sharing

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">âœ…</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">ðŸ”¨</span> Latest commit | 816b858 |
 |<span aria-hidden="true">ðŸ˜Ž</span> Deploy preview | http://PR-373.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/integrate_with_central/api_server.md [hunk 1]
```diff
@@ -202,3 +202,210 @@ attributes:
...
+For the subject, you can use '*' for giving access to all team or use the teamId to specify specific team
+{{< /alert >}}
+
+#### ACL and scoped objects
+
+When object is scoped to another, the sharing of the depending objects is not automatic.
+
+For instance API Service is scoped to environment. One can decide to share his environment with another team. That does not imply that the other team will gain access to the API of the environment.
+
+This should be an explicit will of the owner of the object.
```
Comment (robertengaxway) [inline content/en/docs/integrate_with_central/api_server.md:302]: I believe the precedence here should be explained as well as I remember it, feel free to correct:   
"If an API Service object has no owner defined, the ownership of the scoped environment will be applied to the API Service object.    
If an API Service object has an owner defined, this takes precedence over the ownership of the scoped environment."

**Comment by lbadenhop** [comment]
@cbordier, please approve my changes.

**Comment by cbordier** [comment]
@lbadenhop I approve your changes

**Pull request merged**